corehq/apps/app_manager/views/releases.py

```diff
@@ -170,6 +171,8 @@ def release_build(request, domain, app_id, saved_app_id):
     app_post_release.send(Application, application=saved_app)
 
     if is_released:
+        if saved_app.build_profiles and domain_has_privilege(domain, privileges.BUILD_PROFILES):
```

Comment:
Is it possible to test if all the builds have already been created? although I suppose not many have build profiles